### PR TITLE
INTLY-2102 execute all-tests-executor within the Delorean pipeline

### PR DESCRIPTION
**Pull request by tremes**
See the run https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/OpenShift/job/openshift-cluster-integreatly-test-tremes/24 which was pretty good (only smoke tests failed - I'll investigate it)

### CODE REVIEW

File: jobs/integr8ly/all-tests-executor.yaml [hunk 1]
```diff
@@ -150,7 +150,7 @@
         }
 
         def runTests(testPipeline) {
-            buildStatus = build(job: testPipeline, propagate: false, parameters: parameters).result
+            buildStatus = build(job: testPipeline, propagate: false, parameters: jobParams).result
             println "Build finished with ${buildStatus}"
```
Comment (tremes) [inline jobs/integr8ly/all-tests-executor.yaml]: These both changes are not necessary (just leftover after my experiments) on the other hand this is little bit more clear.

**Comment by pawelpaszki** [comment]
> https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/OpenShift/job/openshift-cluster-integreatly-test-tremes/24

I think smoke tests failed, because there is no Gitea NS

File: jobs/integr8ly/all-tests-executor.yaml (hunk 1 above)
Comment (pawelpaszki) [reply jobs/integr8ly/all-tests-executor.yaml]: I don't mind either way

**Comment by tremes** [comment]
@pawelpaszki yeah I know :)

**Comment by pawelpaszki** [comment]
@tremes I think it is deliberately not installed on OSD as well as a couple of other namespaces. Anyway - LGTM

**Comment by tremes** [comment]
@pawelpaszki yes I think you're right. I'll check. Thanks for review anyway.

**Comment by trepel** [comment]
@tremes Yes, some namespaces are not installed on OSD on purpose. We should be tracking which ones in our test plan.

**Pull request merged**